### PR TITLE
Register Attendee

### LINKED ISSUE #73: Register Attendee Page
Page Url: /entity/event-attendee/event/:eventId/new
Module: Event
Authority:
- CC Member

Usage: Register as event attendee for event with "eventId"

- [ ] Development
- [ ] Unit Test

### PR DESCRIPTION
**Pull request by junzai97**
Resolved #73

### CODE REVIEW

File: src/main/webapp/app/entities/event-attendee/event-attendee-create.tsx [hunk 1]
```diff
@@ -11,37 +12,39 @@ import { IRootState } from 'app/shared/reducers';
...
     this.state = {
-      isNew: !this.props.match.params || !this.props.match.params.id
+      event: null
     };
   }
 
+  getEvent = async () => {
+    const event = await axios.get<IEvent>(`api/events/${this.props.match.params.eventId}`);
+    this.setState({ event: event.data });
+  };
```
Comment (HowardYaw) [inline src/main/webapp/app/entities/event-attendee/event-attendee-create.tsx:35]: can have an enhancement to put this in reducer?

File: src/main/webapp/app/entities/event-attendee/event-attendee-create.scss [hunk 2]
```diff
@@ -0,0 +1,4 @@
+.transport-checkbox {
+  height: 25px !important;
+  margin-top: 0px;
+}
```
Comment (HowardYaw) [inline src/main/webapp/app/entities/event-attendee/event-attendee-create.scss]: maybe can put inside event-module.scss?

File: src/main/webapp/app/routes.tsx [hunk 3]
```diff
@@ -31,7 +31,7 @@ const Routes = () => (
       <ErrorBoundaryRoute exact path="/auth/email/reset" component={AuthEmailReset} />
       <PrivateRoute exact path="/profile" component={UserProfile} />
       <PrivateRoute path="/admin" component={Admin} hasAnyAuthorities={[AUTHORITIES.ADMIN]} />
-      <PrivateRoute path="/entity" component={Entities} hasAnyAuthorities={[AUTHORITIES.USER]} />
+      <ErrorBoundaryRoute path="/entity" component={Entities} />
```
Comment (HowardYaw) [inline src/main/webapp/app/routes.tsx]: can change back to PrivateRoute?

**Review by HowardYaw — DISMISSED** [review]

File: src/main/webapp/app/entities/event-attendee/event-attendee-create.tsx (hunk 1 above)
Comment (junzai97) [reply src/main/webapp/app/entities/event-attendee/event-attendee-create.tsx:35]: Sure, created an enhancement issue in #103

File: src/main/webapp/app/entities/event-attendee/event-attendee-create.scss (hunk 2 above)
Comment (junzai97) [reply src/main/webapp/app/entities/event-attendee/event-attendee-create.scss]: Resolved in fe845d284e2cb16f075baa6dbb9d9c1eba3efa68

File: src/main/webapp/app/routes.tsx (hunk 3 above)
Comment (junzai97) [reply src/main/webapp/app/routes.tsx]: Resolved in fe845d284e2cb16f075baa6dbb9d9c1eba3efa68

**Review by CarlosSia — APPROVED** [review]

**Review by LUXIANZE — APPROVED** [review]

**Pull request merged**